frontend/src/components/common/NullScreen.js

```diff
@@ -107,6 +103,19 @@ const NullScreen = (props) => (
         <p>Create a contact using the menu above</p>
       </div>
     )}
+    {props.type === "404" && (
```

Comment:
Do we have page 'types' elsewhere in the app? This feels like it should be a constant, based on our existing constants patterns.

Reply:
the `type` is for the null screen, not a page `type`

I felt that 404 explained the type of null screen the best, happy to change the string to something else. 

If you expand this file, you'll see the other types being displayed.

Reply:
I agree that 404 is an accurate description. Anything that feels like a magic number or magic string, though, is probably good to question. A typo on this wouldn't get caught by a linter, for example.

But creating a constant called `NOT_FOUND`, for example, can be picked up by a linter and reused (if appropriate) elsewhere.